File: eVuMax.UI/evumax-web-app/src/components/Widgets/AdvKPI/AdvKPIDataFilter.ts
export enum kpiFilterType{
AllData = 0,
LastHours = 1,
DateRange = 2,
DateOnwards = 3,
DepthRange = 4,
DepthOnwards = 5,
CurrentSection = 6
}
export class ADVKPIDataFilter{
    public FilterData:boolean = false;
    public FilterMainWellID: string  = "";
    public FilterType:kpiFilterType  = kpiFilterType.LastHours;
    public Filter_FromDate: Date  = new Date();
    public Filter_ToDate: Date  = new Date();
    public Filter_FromDepth:number = 0;
    public Filter_ToDepth:number  = 0;
    public Filter_LastHours:number  =24;

}

File: eVuMax.DataBroker/AdvKPI/AdvKPIDataFilter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eVuMax.DataBroker.AdvKPI
{

   public enum kpiFilterType
    {
        AllData = 0,
        LastHours = 1,
        DateRange = 2,
        DateOnwards = 3,
        DepthRange = 4,
        DepthOnwards = 5,
        CurrentSection = 6
    }
    public class AdvKPIDataFilter
    {
        public  bool FilterData  = false;
        public string FilterMainWellID  = "";
        public kpiFilterType FilterType  = kpiFilterType.LastHours;
        public DateTime Filter_FromDate  = new DateTime();
        public DateTime Filter_ToDate  = new DateTime();
        public int Filter_FromDepth = 0;
        public int Filter_ToDepth= 0;
        public int Filter_LastHours  =24;
    }
}
